### DISCUSSION TITLE
Is there an object type that is homologous to array<T>?

### DISCUSSION THREAD

**Discussion by rzvc** [vimeo/psalm#7998] [Q&A] [answered]
Like the title says, I would like to be able to declare an object whose properties are all of one type T.

Is this a built in type, or will I have to define it by making a class with __get()?

Thanks

**Comment by orklah** [comment, marked as answer]
You'll have to create an object in any way, and if your property are dynamic, maybe https://psalm.dev/docs/running_psalm/configuration/#universalobjectcrates could help

**Reply by rzvc** [reply]
Already solved this one. Thanks.

Yeah, I ended up creating an object.
